src/controllers/DevicesController.js

```diff
@@ -106,13 +106,14 @@ class DevicesController extends Controller {
 
       return this.ok({ name: updatedAttributes.name, ok: true });
     }
-    // TODO not implemented yet
     // 2 flash device with known app
     if (postBody.app_id) {
-      this._deviceRepository.flashKnownApp(
+      await this._deviceRepository.flashKnownApp(
         deviceID,
+        this.user.id,
         postBody.app_id,
```

Comment:
i don't like `postBody.app_id`, it should be `appName`, but I saw it somewhere in old code..so was thinking that how they send `known_app` fileName..but as I sad seems particle-cli don't have an ability to flash with some server-side known_app...Maybe it actually does and I just missing something...so..left this as it is for now.